director/commands/user.py

```diff
@@ -0,0 +1,69 @@
+from werkzeug.security import generate_password_hash
+from terminaltables import AsciiTable
+
+
+from director.context import pass_ctx
+from director.exceptions import UserNotFound
+from director.models.users import User
+
+import click
+
+
+def _get_users():
+    return User.query.all()
+
+
+@click.group()
+def user():
+    """Manage the users"""
+
+
+@user.command(name="list")
+@pass_ctx
+def list_users(ctx):
+    """Display users"""
+    data = [["Username"]]
+    users = _get_users()
+
+    for user in users:
+        data.append([user.username])
+
+    table = AsciiTable(data)
+    table.inner_row_border = True
+    click.echo(table.table)
+
+
+@user.command(name="create")
+@click.argument("username")
+@click.argument("password")
```

Comment:
Can you use the password prompt provided by click plz ? https://click.palletsprojects.com/en/7.x/options/#password-prompts